.travis/lint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+# linting
+
+go get -u honnef.co/go/tools/cmd/megacheck
+go get -u golang.org/x/lint/golint
+go get -u golang.org/x/tools/cmd/goimports
+
+# go 1.8 doesn't exclude /vendor/ yet.
+dirs=$(go list -f {{.Dir}} ./... | grep -v /vendor/)
+pkgs=$(go list ./... | grep -v /vendor/)
```

Comment:
I assume this could go away now

Reply:
you mean the `grep -v /vendor/` part? Looks like

Reply:
exactly.

Reply:
removed